ivadomed/scripts/prepare_dataset.py

```diff
@@ -0,0 +1,146 @@
+import argparse
```

Comment:
I would recommend to rename this script as this is very specific to vert labelling task, something like prepare_dataset_vertebral_labelling?